Vagrantfile

```diff
@@ -242,4 +242,23 @@ Vagrant.configure("2") do |config|
     hosts_config.add_host "#{ENV['JENKINS_MASTER_IP_ADDRESS']}", ["#{ENV['JENKINS_MASTER_URL']}"]
     hosts_config.add_host "#{ENV['DOCKER_SLAVE_IP_ADDRESS']}", ["#{ENV['DOCKER_SLAVE_URL']}"]
   end
+
+ config.vm.define "kali-linux" do |kali_linux|
```

Comment:
Sorry, there is just one more thing. In line with the other machine definitions, can you change the "kali-linux" part to match the machine name used in the Makefile target please? So it would be the name of the target just less the `vm-` prefix and the `vbox` suffix. Don't mind if it's in a separate commit.